src/components/Table.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {cn} from "../utils/util";
+
+
+export interface TableColumn {
+    field: string;
+    headerName: string;
+    width: number;
+    className?: string;
+}
+
+export interface TableRow {
+    [key: string]: React.ReactNode;
+}
+
+interface CardProps {
+    rows: TableRow[];
+    columns: TableColumn[];
+    className?: string;
+}
+
+export function Table({rows, columns, className}: CardProps) {
+    return (
+        <div className={cn("flex h-fit max-h-full w-full flex-col overflow-auto bg-gray-875", className)}>
+            <div className="flex h-fit w-max min-w-full flex-col">
+                <div className="sticky top-0 flex h-6 min-h-[24px] items-center gap-2.5 self-stretch overflow-x-hidden border-b border-b-fade-700 bg-gray-875 px-3">
+                    {columns.map((column, cIndex) => {
+                            return (
+                                <div className="" key={cIndex} style={{width: column.width}}>
```

Comment:
please remove the empty classNames prop.
Also regarding width, It might be better to use flex instead, so each column gets a certain percentage/weight of the total width?